### PR TITLE
[LYNX-90] enable working example with download

### PR DESCRIPTION
**Pull request by lecstor**
- implement `npm run example:export`
- fix main export Babel6 issue
- add function to enable downloading export files

### CODE REVIEW

File: src/api.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import rp from 'request-promise';
+import request from 'request';
```
Comment (koresar) [inline src/api.js:2]: Why we need `request` here in addition to `request-promise`?

Comment (lecstor) [inline src/api.js:2]: the download method I added streams the download to the file. `request-promise` recommends not using it for streaming but rather using plain `request`.

Comment (koresar) [inline src/api.js:2]: :+1:

**Comment by johndagostino** [comment]
made some changes, merging

**Pull request merged**